python/src/WeibullMinFactory_doc.i.in

```diff
@@ -8,7 +8,7 @@ DistributionFactory, WeibullMin
 Notes
 -----
 Several estimators to build a WeibullMin distribution from a scalar sample
-are proposed.
+are proposed. The default strategy is using the maximum likelihood estimators.
```

Comment:
```suggestion
are proposed. The default estimator is the maximum likelihood estimator.
```

Reply:
I really think that we can keep the first formulation that says the same thing as the formulation you propose. All the factory classes have been documented with this sentence.

Reply:
Ok, this is minor, no need to change everything.